akka-actor/src/main/scala/akka/io/DirectByteBufferPool.scala

```diff
@@ -71,6 +71,29 @@ private[akka] class DirectByteBufferPool(defaultBufferSize: Int, maxPoolEntries:
       tryCleanDirectByteBuffer(buf)
   }
 
+  private final def tryCleanDirectByteBuffer(toBeDestroyed: ByteBuffer): Unit = DirectByteBufferPool.tryCleanDirectByteBuffer(toBeDestroyed)
+}
+
+/** INTERNAL API */
+private[akka] object DirectByteBufferPool {
```

Comment:
could we even move this to ` akka.io.DirectByteBufferPool`, which has a similar thing?